worker/src/cf.rs

```diff
@@ -150,7 +150,8 @@ impl Cf {
         self.inner.region_code()
     }
 
-    /// Timezone of the incoming request
+    /// **Requires** `timezone` feature. Timezone of the incoming request
+    #[cfg(feature = "timezone")]
     pub fn timezone(&self) -> impl chrono::TimeZone {
```

Comment:
I think the return type here should be changed to `Option<impl chrono::TimeZone`. If the user directly depends on `chrono-tz` with the `filter-by-regex` feature then parsing will fail if the time zone has been filtered out.

Reply:
I wonder if this function should be removed altogether. It returns an opaque type that's not usable in any way without bringing in another dependency. At that point you might as well leave the time zone parsing to the user.

Reply:
Does this change enable you to actually use the filter, or should I also enable that filter in `worker`?

Reply:
The filter works as long as somewhere in the dependency graph the `filter-by-regex` feature is enabled and the environment variable is passed to cargo. This is outside the control of the `worker` create so it should not assume all time zones are available.

Reply:
Ok, I've changed this method to return a `Result`